bundler.d/dynflow_sidekiq.rb

```diff
@@ -1,5 +1,5 @@
 group :dynflow_sidekiq do
-  gem 'sidekiq', '~> 5.0'
+  gem 'sidekiq', '~> 6.3.0'
   gem 'gitlab-sidekiq-fetcher', require: false
   gem 'sd_notify', '~> 0.1'
```

Comment:
This is slightly troublesome. We require that there is a single consumer (1 process, concurrency level 1) of dynflow_orchestrator queue to be able to enforce ordering. In case multiple instances are started, we ensure only one is active by placing a lock into redis, the instances which failed to acquire the lock will be passive, acting as hot spares.

I must admit, this is currently broken even if we have control over when systemd is notified (we do it too late) so no real change here. What sidekiq does also happens too late.

But yes, we should be able to rip out the dependency but fixing the issue mentioned above will be slightly more difficult.

Reply:
So you're saying it can remain in a loading state until the lock is gone? Would it make sense to send a `READY=1` and immediately a `RELOADING=1` to pretend it started, but then show it really isn't doing anything? No clue if this works.

Reply:
Yes, that's what I'm saying. Not sure if that workaround would work, also no idea how systemd expects watchdog to work and if we'd have to somehow simulate that as well

Reply:
We don't set a watchdog (note how https://github.com/theforeman/foreman/blob/develop/extras/systemd/dynflow-sidekiq%40.service is missing such a statement). However, there is a timeout if you don't start up in time and the service will be killed.

Reply:
Or just send `READY=1` and `STATUS=Standing by` inside dynflow-sidekiq.rb before trying to acquire the lock, then doing `STATUS=Active` when the lock is acquired and then letting sidekiq do whatever it wants?

Reply:
Sending a more explicit status is also fine by me.